src/init.c

```diff
@@ -1290,7 +1306,8 @@ int main(int argc, char *argv[])
 		return -1;
 	}
 
-	if (mount("devpts", "/dev/pts", "devpts", MS_NOSUID| MS_NOEXEC, NULL) == -1) {
+	if (system_mount("devpts", "/dev/pts", "devpts", MS_NOSUID| MS_NOEXEC,
+			 NULL) == -1) {
 		perror("mount devpts failed");
 		return -1;
 	}
```

Comment:
I think we could move the above code and some init-related code into a function.
and it should be called only when argv[0] == "init".